client/app/scripts/selectors/chartSelectors.js

```diff
@@ -95,6 +96,12 @@ export const dataNodesSelector = createSelector(
 );
 
 
+export const searchableFieldsSelector = createSelector(
```

Comment:
Shouldnt this be a selector that checks for deep equality (fields are always new sets)?